data/50unattended-upgrades.Ubuntu

```diff
@@ -11,6 +11,8 @@ Unattended-Upgrade::Allowed-Origins {
 	// available, the policy for updates is such that unattended-upgrades
 	// should also install from here by default.
 	"${distro_id}ESM:${distro_codename}";
+	"${distro_id}ESM:${distro_codename}-apps-security";
```

Comment:
Thanks for tackling this @rbalint, my fault on the bug file typo. ESM for apps has a different origin (UbuntuESMApps) than ESM for infra (UbuntuESM).

I have updated the bug description and title.
https://bugs.launchpad.net/ubuntu/+source/unattended-upgrades/+bug/1857051 

So I think this line should be
"${distro_id}ESMApps:${distro_codename}-apps-security";